server.js

```diff
@@ -6345,17 +6345,11 @@ cache(function(data, match, sendBadge, request) {
   var url = 'https://registry.hub.docker.com/v2/repositories/' + path + '/buildhistory';
   var badgeData = getBadgeData('docker build', data);
   request(url, function(err, res, buffer) {
-    if (err != null) {
-      badgeData.text[1] = 'inaccessible';
+    if (checkErrorResponse(badgeData, err, res)) {
       sendBadge(format, badgeData);
       return;
     }
     try {
```

Comment:
It's helpful to tell the user that it's the repo. How about adding an optional parameter to the helper function to allow a badge-specific "not found" message to be used? (e.g. like [in this WIP](https://github.com/badges/shields/compare/node-8...paulmelnikow:not-found#diff-7f07515da2c344002004944a4806b857R23))